x-pack/plugins/ml/public/application/datavisualizer/data_comparison/data_drift_index_patterns_editor.tsx

```diff
@@ -0,0 +1,370 @@
+                    {
+                      defaultMessage: 'Select an optional timestamp field',
+                    }
+                  )}
+                  singleSelection={{ asPlainText: true }}
+                  options={combinedTimeFieldOptions}
+                  selectedOptions={timeField}
+                  onChange={(newValue) => {
+                    if (newValue.length === 0) {
+                      // Don't allow clearing the type. One must always be selected
+                      return;
+                    }
+                    setTimeField(newValue);
+                  }}
+                  isClearable={false}
+                  isDisabled={productionIndexPattern === '' && referenceIndexPattern === ''}
+                  data-test-subj="timestampField"
+                  aria-label={i18n.translate(
+                    'xpack.ml.dataDrift.indexPatternsEditor.timestampSelectAriaLabel',
+                    {
+                      defaultMessage: 'Timestamp field',
+                    }
+                  )}
+                  fullWidth
+                />
+              </>
+            </EuiFormRow>
+          ) : null}
+          {!foundDataViewId ? (
+            <EuiFormRow
+              label={i18n.translate('xpack.ml.dataDrift.indexPatternsEditor.dataViewName', {
+                defaultMessage: 'Data view name',
+              })}
+              helpText={`Optional data view name. ${
+                referenceIndexPattern && productionIndexPattern
+                  ? `Default to '${referenceIndexPattern},${productionIndexPattern}' if not set.`
+                  : ''
+              }`}
+              fullWidth
+            >
+              <EuiFieldText
+                value={dataViewName}
+                onChange={(e: ChangeEvent<HTMLInputElement>) => {
+                  setDataViewName(e.target.value);
+                }}
+                fullWidth
+                data-test-subj="dataDriftDataViewNameInput"
+                placeholder={`Example Name`}
+              />
+            </EuiFormRow>
+          ) : null}
+
+          {dataViewMsg ? <EuiCallOut color="primary">{dataViewMsg}</EuiCallOut> : null}
+
+          <EuiFormRow id="analyzeDriftData">
+            <EuiFlexGroup>
+              {canEditDataView && foundDataViewId === undefined ? (
+                <EuiFlexItem>
+                  <EuiButton
+                    color="primary"
+                    disabled={!productionIndexPattern || !referenceIndexPattern}
+                    onClick={createDataViewAndRedirectToDataComparisonPage.bind(null, true)}
+                    iconType="visTagCloud"
+                    data-test-subj="analyzeDataDriftButton"
+                    aria-label={i18n.translate(
+                      'xpack.ml.dataDrift.indexPatternsEditor.analyzeDataDriftWithoutSavingLabel',
+                      {
+                        defaultMessage: 'Analyze data drift without saving',
```

Comment:
We are mixing the terminology here, with the view called `Data comparison` but the term `Data drift` being used for these controls. What are your thoughts on using `Compare without saving` and `Compare` for these controls?

Reply:
Resolving comment as the rename of the view to `Data Drift` means the terminology is now consistent.